jobs/JHAFS_MERGE

```diff
@@ -75,6 +82,8 @@ else
     export RESTARTsrc=${WORKhafs}/intercom/RESTART_analysis
   elif [ -e ${WORKhafs}/intercom/RESTART_vi ]; then
     export RESTARTsrc=${WORKhafs}/intercom/RESTART_vi
+  elif [ -e ${WORKhafs}/intercom/RESTART_init ]; then
```

Comment:
@lgramer remind me, is this change to properly read the init RESTART files to update the moving nests that are initialized in the forecast parent domain?